### ISSUE TITLE
fs.writeFile() crashes renderer process

### ISSUE THREAD

**Issue by yishn** [electron/electron#5016]
* Electron version: v0.37.4
* Operating system: Windows 10 x64

When writing a file using the `fs` module, the renderer process crashes. This doesn't happen in Electron v0.37.3. To reproduce this, add the following lines in `index.html` of the default app:

~~~html
<script>
var contents = []
var max = 10

for (var i = 0; i < max; i++) {
    contents.push('Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet.')
}

require('fs').writeFileSync('./blah', contents.join('\n'))
</script>
~~~

Oddly enough, when setting `max = 5` the renderer process doesn't crash.

**Comment by hunterjm** [comment]
I can confirm that this also occurs in v0.37.4 on Mac El Capitan with the async version of `fs.writeFile`.  The issue is not present in v0.37.3.

**Comment by erkyrath** [comment]
Yup, I ran into this too.

**Comment by feross** [comment]
Duplicate of https://github.com/electron/electron/issues/5028 and now fixed in `master`.

**Comment by yishn** [comment]
Fixed in v0.37.5
